server.js

```diff
@@ -175,12 +179,14 @@ app.locals.LOGOS_ORIGIN = AppConstants.LOGOS_ORIGIN;
 app.locals.UTM_SOURCE = new URL(AppConstants.SERVER_URL).hostname;
 
 const SESSION_DURATION_HOURS = AppConstants.SESSION_DURATION_HOURS || 48;
+const SESSION_COOKIE_SECURE = AppConstants.NODE_ENV === "dev" ? false : true;
```

Comment:
This works, or we could do `AppConstants.NODE_ENV !== "dev"` since that's already a Boolean.

Reply:
I like it! Updated.